buildlib/jucx/jucx-test.yml

```diff
@@ -20,13 +21,11 @@ jobs:
 
     steps:
       - checkout: self
-        fetchDepth: 100
         clean: true
         displayName: Checkout
       - bash: |
           set -xeE
           source buildlib/az-helpers.sh
-          check_gpu ${{ parameters.name }}
```

Comment:
can we still check that gpu is present when we use ucx_gpu machine?

Reply:
Of course. Merged from master and got it back.